.golangci.yaml

```diff
@@ -76,7 +75,6 @@ linters:
     - unconvert
     - unused
     - usestdlibvars
-    - varnamelen
```

Comment:
[this is an annoying rule](https://github.com/blizzy78/varnamelen); i can understand the original motivation, but in practice, i'd like to use short variable names like `id`